lib/react_on_rails/helper.rb

```diff
@@ -512,6 +570,9 @@ def server_rendered_react_component(render_options)
                                                js_code: js_code)
       end
 
+      # TODO: handle errors for streams
+      return result if render_options.stream?
```

Comment:
The ability to show the error details happen while rendering a streamable component will be handled in a follow up PR